File: 4248-count-commas-in-range-ii/count-commas-in-range-ii.java
class Solution {
    public long countCommas(long  n) {
        if(n<1000) return 0;
        long temp=n;
        long base=1000;
        long commasUsed=0;
        while(base<=temp){
            commasUsed+=(temp-base+1);
            base*=1000;
        }
        return commasUsed;
    }
}

File: 4248-count-commas-in-range-ii/count-commas-in-range-ii.py
class Solution:
    def countCommas(self, n: int) -> int:
        if (n<1000) : return 0
        temp = n
        base = 1000
        ans = 0
        while(base<=temp):
            ans+=temp-base+1
            base*=1000
        return ans
